docs/migration-guide.md

```diff
@@ -0,0 +1,330 @@
+
+Switch to `d2l-test-runner` from `web-test-runner`/`wtr` binaries in any `package.json` scripts.
+
+By default, `d2l-test-runner` will:
+
+* Run in Chrome, Firefox and Safari (except `vdiff` just runs in Chrome)
+* Enable the `--node-resolve` flag
+* Look for tests in `./test/**/*.test.js` (using the `test` group)
+* Suppress `ResizeObserver` errors (a common reason repos use a config file)
+
+Due to these new defaults, if the repo is using a `web-test-runner.config.js`, it may no longer be necessary and could be removed. If the config file is necessary, rename it to `d2l-test-runner.config.js`.
+
+Before
+
+```yml
+  "scripts": {
+    "test:unit": "web-test-runner --files \"./test/**/*.test.js\" --node-resolve"
+  }
+```
+
+After:
+
+```yml
+  "scripts": {
+    "test:unit": "d2l-test-runner"
+  }
+```
+
+## vdiff Tests
+
+Breaking changes were made between the `@brightspace-ui/visual-diff` library to `@brightspace-ui/testing`'s vdiff functionality. These changes will require a migration, some of which can be done automatically but unfortunately some is manual.
+
+Overview of changes:
+
+* File/directory name changes (automatic migration)
+  * Suite names which were previously repeated in the directory and filename have been removed
+  * Leading `d2l-` in suite names has been removed
+  * Browser name (always `chromium` previously) is now included in the path
+  * Local snapshots are now stored in a `.vdiff` directory at the root of the project (which should be listed in `.gitignore`)
+  * CI goldens are still stored alongside the tests in a `golden` directory, but without `screenshots/ci`
+* Fixtures were defined HTML files and tests in JavaScript files. Both are now defined in JavaScript. (manual migration)
+* Puppeteer was previously used to interact with the browser. It has been replaced with Playwright, but the browser can now be interacted with directly. (manual migration)
+
+### Strategy for Large Repositories
+
+For repositories with many vdiff tests, it may be easier to migrate a subset of tests at a time.
+
+By working in this way, both the old and new vdiff libraries continue to run in parallel as tests are gradually migrated from one to the other. Then once all tests migrated, the old infrastructure can be removed.
+
+The automated migration tools described below support an optional file pattern to make a gradual migration process possible.
+
+### Step 1: Automatically Migrate Local Goldens
+
+Before migrating a set of tests, it's helpful to first generate a local set of goldens using the old `@brightspace-ui/visual-diff` library.
+
+```bash
+d2l-test-runner vdiff migrate-local ./**/<location>/*
+```
+
+The optional location parameter can be used to migrate a subset of tests.
+
+This will:
+
+1. Install the old `@brightspace-ui/testing` library
+2. Run the old library to generate local goldens
+3. Optional: pass a location to run a subset of tests
+4. Move the local goldens to their new location in the `.vdiff` directory
```

Comment:
Maybe combine 2 and 3?

```suggestion
2. Run the old library to generate local goldens (optionally, pass a location to run a subset of tests rather than all of them)
3. Move the local goldens to their new location in the `.vdiff` directory
```